pom.xml

```diff
@@ -255,26 +229,6 @@
                     <target>1.8</target>
                 </configuration>
             </plugin>
-            <plugin>
```

Comment:
Do not remove jacoco. It is used to calculate code coverage.

Reply:
This was there twice in the pom. I removed one with older version